corehq/apps/userreports/models.py

```diff
@@ -206,14 +227,40 @@ def save(self, **params):
     def data_source_id(self):
         return self._id
 
-    def filter(self, document):
+    def filter(self, document, eval_context=None):
```

Comment:
`get_items` in this class could also pass in `eval_context` to `self.filter`

https://github.com/dimagi/commcare-hq/pull/23799/files#diff-3af365e79b93f051d416585bcba61b7eR414

Reply:
https://github.com/dimagi/commcare-hq/pull/23799/commits/e1f5757ee11cf7bd0fa3f7e00616ceac59711b52